Src/BootCamp.Chapter/LinqMethodDemo.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    public static class LinqMethodDemo
+    {
+        internal static IEnumerable<T> Demo<T>(this IEnumerable<T> list, Predicate<T> predicate)
```

Comment:
I wouldn't call this demo, but ``SnapFingers`` or RemoveHalfOfAllElementsAtRandom()

Reply:
resolved